network/edge-services/reference-content/understanding-pricing.mdx

```diff
@@ -0,0 +1,169 @@
+
+For example:
+
+<Message type="important">
+The example prices and limits used below are subject to change. You should always refer to the [pricing page](https://www.scaleway.com/en/pricing/network/#edge-services) for the most up to date information.
+</Message>
+
+- From November 1-10 you are subscribed to the **Starter** plan.
+- On November 11, you switch to the **Professional** plan.
+- You remain subscribed to the **Professional** plan for the rest of the month (ending November 30)
+
+There are 30 days in the month of November, and you spent 10 of them subscribed to the Starter plan, and 20 of them subscribed to the Professional plan. Your November Edge Services billing, in terms of the monthly plan price, is therefore calculated as follows:
+
+`({Monthly price for Starter plan} / 30 days) * 10 days` + `({Monthly price for Professional plan} / 30 days) * 20 days` <br/>
+e.g. `(0.99 / 30) * 10 = 0.33` + `(12.99 / 30) * 20 = 8.66 ` = `€8.99`
+
+### Pipeline charges
+
+Any additional pipeline charges accumulated when you had more pipelines in existence than the previous plan's limit will remain on your monthly bill. From the moment you change your plan, you can consume pipelines up to the new plan's monthly limit without being charged.
+
+Additional pipeline charges are based on a monthly fee per additional pipeline. This monthly fee is applied pro-rata based on how long you had additional pipelines in existence during the calendar month.
+
+For example:
+
+<Message type="important">
+The example prices and limits used below are subject to change. You should always refer to the [pricing page](https://www.scaleway.com/en/pricing/network/#edge-services) for the most up to date information.
+</Message>
+
+- From November 1-10 you are subscribed to the **Starter** plan, which has a limit of 1 pipeline.
+- You have 2 pipelines in existence during these 10 days.
+- On November 11, you upgrade to the **Professional** plan, which has a limit of 10 pipelines.
+- You create 8 more pipelines, which you keep for the rest of the month: you finish the month with 10 pipelines.
+
+There are 30 days in the month of November. For 10 of these days, you had an extra pipeline beyond the limits of your **Starter** plan. After upgrading, and even after creating more pipelines, you were within the pipeline limits of your new plan. Your November Edge Services billing, in terms of the additional pipeline charges, is therefore calculated as follows:
+
+`({Monthly flat fee for an additional pipeline} / 30 days) * 10 days` <br/>
+e.g. `(4 / 30) * 10 = €1.33`
+
+### Cache charges
+
+Any additional cache charges accumulated when you exceeded your previous plan's cache limit will remain on your monthly bill. From the moment you change your plan, your cache consumption resets to 0. During the rest of the month, you can consume cache up to the new plan's limit without being charged.  
+
```

Comment:
It's also worth telling that if there is plan change, the extra cache (if any) will added on top of the cache for the "lowest" plan.
So:
-  if a downgrade occurs: all the cached data within the hour will be added to the cache for the new plan
- if an upgrade occurs: all the cached data within the hour will be added to the cache for the old plan

We have to do it since technically speaking, it's a tricky use case